controllers/default.py

```diff
@@ -71,9 +76,11 @@ def api():
     return Collection(db).process(request,response,rules)
 
 def register():
+    auth.settings.register_next = 'confirm.html'
     response.title = 'iDibo Registration'
```

Comment:
I tried to do that but the code in layout.html overides it all, so i just left it simply at changing the title in the controller function for the pages as pointed out in the other comment below